ruby-challenges.rb

```diff
@@ -2,29 +2,150 @@
 # MINASWAN
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter. Use the test variables provided.
+# Psuedocode: What do I want to do ?Return an array of all the words containg a particual letter. How do I do this ? create a method called result that takes in an array of words and a single letter. How will this method accomplish this ? : the Result method will be called by using the select method on beverage_array, I will use the select method to locates and returns elements in the array that matches a condition I specify such as containing the letter 'o' or 't' . .The select method  constructs a new array containing all of the elements that match the condition, instead of just returning a single value and stopping.
 
-beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+# beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
 letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
-
-
+result = beverages_array.select {|item| item.include?(letter_t)}
```

Comment:
This is a great start! Can we put this logic into a method and make it dynamic? You have the idea in your pseudocode, create a method that takes in an array and a single letter so I am thinking 2 parameters.